igtdetect/glossharvester.py

```diff
@@ -0,0 +1,99 @@
+
+def get_utterance(row: str):
+    return row.split(':')[1].strip('\n') if ':' in row else 'NA'
+
+def get_linenr(row: str):
+    return row.split('line=')[1].split()[0] if 'line=' in row else 'NA'
+
+def get_linetag(row: str):
+    return row.split('tag=')[1][0] if 'tag=' in row else 'NA'
+
+def get_iscore(row:str):
+    return float(row.split('iscore=')[1][0:3]) if 'iscore' in row else 0
+
+class IGT():
+    def __init__(self, line="NA", gloss="NA", translation="NA", source="NA", linenr=0, classification_methods=[]):
+        self.line = line
+        self.gloss = gloss
+        self.translation = translation
+        self.source = source
+        self.linenr = linenr
+        self.classification_methods = classification_methods
+
+    def __str__(self):
+        return f"Source: {self.source}\nL: {self.line}\nG: {self.gloss}\nT: {self.translation}\nClassification methods: {self.classification_methods}\n"
```

Comment:
would be nice to also store the original page and line number, in case you want to look back at the pdf for debugging purposes